docs/mergeduty/howto.md

```diff
@@ -1,102 +1,137 @@
 # MergeDuty
 
 Most code changes to Firefox and Fennec land in the [mozilla-central](https://hg.mozilla.org/mozilla-central) repository, and 'nightly' releases are built from that repo. 
-DevEdition, Beta and Release Candidate releases are built from the [mozilla-beta](https://hg.mozilla.org/releases/mozilla-beta/) repository, Extended Support Releases from the relevant ESR repo, such as [mozilla-esr52](https://hg.mozilla.org/releases/mozilla-esr52/), and main releases are created from [mozilla-release](https://hg.mozilla.org/releases/mozilla-release/).
+DevEdition and Beta are built from the [mozilla-beta](https://hg.mozilla.org/releases/mozilla-beta/) repository, Extended Support Releases from the relevant ESR repo, such as [mozilla-esr52](https://hg.mozilla.org/releases/mozilla-esr52/), and Release and Release Candidates are built from [mozilla-release](https://hg.mozilla.org/releases/mozilla-release/).
 
 How are those repositories kept in sync? That's MergeDuty. You will be merging repositories to create new beta, release candidates, releases and extended support releases.
 
-# Overview
-MergeDuty consists of multiple separate days of work. Each day you must perform several sequential tasks. The days are spread out over nearly three weeks.
-
-The process usually operates like this:
-* A few days before the merge, do the prep work
-* mozilla-beta merges to mozilla-release
-* mozilla-esr gets version bumped
-* mozilla-central merges to mozilla-beta, perhaps several times during the release cycle.
-* Devedition go-to-build
-* mozilla-central merges to mozilla-beta for the last time
-* Firefox Beta go-to-build
+## Overview of Procedure
+
+MergeDuty consists of multiple separate days of work. Each day you must perform several sequential tasks. The days are spread out over nearly three weeks, with 3 major days of activity.
+
+The releng process usually operates like this:
+* A week before the merge, do the prep work
+  * Verify you have access to what you need
+  * Create the buildbot-config version bump patches that will land on Merge day
+  * Make sure that all the dry-run migrations run cleanly
+  * Sanity check you have no blocking migration bugs
+* On Merge day
+  * ask vcs to disable hg.m.o l10n hooks
```

Comment:
Also some instructions for this would be handy for first-time users

Reply:
example of a bug and how-to in the relative section below. This is an overview of bullet points. We should link to the sections below though

Reply:
Already being tracked in issue 76 re: above